googletest/package.xml

```diff
@@ -31,7 +31,7 @@
 
   1.8.9000 = google/googletest@c6cb7e033591528a5fe2c63157a0d8ce927740dc + osrf commits
   -->
-  <version>1.8.9000</version>
+  <version>1.10.9000</version>
```

Comment:
Based on the scheme described above, this should be `1.10.0` right? Or is this `1.10.0` with some extra commits from master (and therefore `1.10.9000`)?

If it's an exact upstream version, then it should be exactly that version. You should only use the `9000` scheme if it's including extra commits beyond the upstream version.

Reply:
After this merge, there will still be commits from us.  At least:

https://github.com/ament/googletest/commit/a10c0d60218d4bc1589edd3f49e2ef8efe197eb9
https://github.com/ament/googletest/commit/fd6cde7ef3392c0fd490a4c0f765c00830a9ea81
https://github.com/ament/googletest/commit/61582f0435a81abfa559206bdd8dc0f5f4641120
https://github.com/ament/googletest/commit/51a1a80a481f1d5eb0ab524ec4a673b7aa05b6ee

Are all custom commits from us.  So I still think the `1.10.9000` makes sense, no?

Reply:
Ok yeah that’s ok too I guess.